src/store/ducks/opensrp/hierarchies/index.tsx

```diff
@@ -547,7 +553,14 @@ export const getCurrentChildren = () => {
  * @param props -  the filterProps
  */
 export const getAllSelectedNodes = () =>
-  createSelector(getTreeWithMeta(), getLeafNodesOnly, computeSelectedNodes);
+  createSelector(
+    getTreeWithMeta(),
```

Comment:
heads up:  getTreeWithMeta will be removed.

Reply:
@p-netm how do you propose we resolve that in this PR?

Reply:
@moshthepitt by merging #1127